packages/web-extension/src/content-scripts/player/components/Player.vue

```diff
@@ -29,10 +29,21 @@
       <notification-center class="as-right-content" />
       <controls-toolbar class="as-bottom-content" />
 
+      <div
+        :class="{
+          'as-absolute as-bottom-20 as-z-3': true,
+          'as-right-80': isTimestampsPanelOpen,
+          'as-right-108': isPreferencesDialogOpen,
+          'as-right-12': !(isTimestampsPanelOpen || isPreferencesDialogOpen),
+        }"
+      >
+        <SkipButton />
+      </div>
+
       <!-- Dialogs -->
       <ScreenshotOverlay />
-      <TimestampsPanel />
-      <PreferencesDialog />
+      <TimestampsPanel v-model:isOpen="isTimestampsPanelOpen" />
+      <PreferencesDialog v-model:isOpen="isPreferencesDialogOpen" />
```

Comment:
@aklinker1 
How do you feel about these `v-model`s?
I am asking because while I think syntactically it is quite a terse way to keep track of when the dialogs are open, it seems to imply that by setting `isTimestampsPanelOpen.value = whatever`, that will actually open/close the dialogue. So, I am not a fan of that but I wanted to get your thoughts on it anyway~

Reply:
Right on, ive removed the events emitted by these dialogs that I added and now use dialogState as you suggested below!